### ISSUE TITLE
Certain ng-template names don't work

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar-angular#426]
`resourceLabelContent` and a few others.

Originally reported in by @oliverguenther in https://github.com/fullcalendar/fullcalendar-angular/issues/204#issuecomment-1342911591:

> I can't seem to get the resourceLabelContent ng-template to work. I couldn't get figure out the routing from the ContentChild to the custom rendering pipeline though to properly debug it. Do you have any pointers on where to look?
>
> I have a minimal repo example here:
> https://stackblitz.com/edit/angular-ivy-wbmq1i?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html

**Comment by arshaw** [comment]
The bug has been fixed in my local dev environment and will be in the next release.

In the meantime, as a workaround for `resourceLabelContent` you can use `cellContent` instead (the wrong name that will for only beta.4)

**Comment by oliverguenther** [comment]
Hi @arshaw, thanks for getting back so quickly. On my app and the minimal repro example, `cellContent` doesn't appear to work either? The ContentChild seems to be present with that name: https://github.com/fullcalendar/fullcalendar-angular/blob/v6.0.0-beta.4/lib/src/full-calendar.component.ts#L53

**Comment by hoangdinhcong** [comment]
Hi @arshaw, thank you for your hard work. I'm very excited to upgrade to version 6.0.0 with this new feature.

But I have some errors when trying to implement the `resourceLabelContent ` and `resourceAreaHeaderContent` when using it in inside a dynamic dialog in **primeng**. When I close the dialog it shows these errors:

```text
ERROR DOMException: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at TransportContainerComponent.ngOnDestroy (https://github-msefaz.stackblitz.io/turbo_modules/@fullcalendar/angular@6.0.0/fesm2015/fullcalendar-angular.mjs:164:24)
    at executeOnDestroys (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:6454:32)
    at cleanUpView (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:6364:9)
    at destroyViewTree (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:6197:17)
    at destroyLView (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:6342:9)
    at ViewContainerRef.remove (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:22721:13)
    at ViewContainerRef.clear (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:22564:18)
    at NgIf._updateView (https://github-msefaz.stackblitz.io/turbo_modules/@angular/common@15.0.3/fesm2015/common.mjs:3528:37)
    at set ngIf [as ngIf] (https://github-msefaz.stackblitz.io/turbo_modules/@angular/common@15.0.3/fesm2015/common.mjs:3495:14)
    at setInputsForProperty (https://github-msefaz.stackblitz.io/turbo_modules/@angular/core@15.0.3/fesm2015/core.mjs:12467:35)
```


I've created a [Example](https://stackblitz.com/edit/github-msefaz?file=src/app/app.component.ts) here. Please take a look. Thank you so much 💪

**Comment by arshaw** [comment]
@oliverguenther, the ng-template bug you originally posted is solved in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0)

@hoangdinhcong, please see this ticket:
https://github.com/fullcalendar/fullcalendar/issues/7058
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?
